os-autoinst-obs-auto-submit

```diff
@@ -232,6 +231,7 @@ is_staging_specified() {
 
 submit() {
     # submit from staging project if specified
+    is_staging_specified && [[ $staging_project != "$src_project" ]] && src_project=$staging_project
```

Comment:
I am wondering if the check `[[ $staging_project != "$src_project" ]]` is even necessary?